### DISCUSSION TITLE
Check for null response

### DISCUSSION THREAD

**Discussion by j0rdan** [lukevella/rallly#386] [Ideas]
I've had a couple of users, enter their name and immediatley hit 'Save', thinking they do this and then respond with dates.

Can we have a check, if no responses to dates, prompt to ask if this is what they intended?

**Comment by lukevella** [comment]
Hi @j0rdan, the new version I'm working on includes changes to the workflow that should help avoid this situation. I'm not spending too much time making updates on the current version since the aim is to replace it.

**Reply by j0rdan** [reply]
Thanks for the update Luke.

Will the update be rolled into this github repository, or will it be a sperate thing?

**Reply by lukevella** [reply]
Yes, it will be rolled out here once it is ready for user testing.
